returnn/torch/engine.py

```diff
@@ -254,11 +254,49 @@ def _load_model(self, epoch):
         self._model = get_model_func(**sentinel_kw)
         assert isinstance(self._model, torch.nn.Module)
 
+        preload_from_files = self.config.typed_value("preload_from_files", {})
         if epoch > 1:
             filename = self.get_epoch_model_filename(epoch=epoch - 1) + util.get_model_filename_postfix()
             print("Load model %s" % (filename,), file=log.v4)
             model_state = torch.load(filename)
             self._model.load_state_dict(model_state)
+        elif preload_from_files:
```

Comment:
This is wrong here. This would only be executed with `epoch == 1`. But this is wrong. It must always be executed.